src/live_restore/live_restore_server.c

```diff
@@ -113,7 +113,8 @@ __live_restore_worker_stop(WT_SESSION_IMPL *session, WT_THREAD *ctx)
              * WiredTiger starts in a post-background migration state, we should call live restore
              * clean up directly instead of spinning up the server to eventually trigger clean up.
              */
-            WT_ERR(__live_restore_clean_up(session, ctx));
+            if (!F_ISSET(S2C(session), WT_CONN_CLOSING))
```

Comment:
Style nit. If we pull the `if (!F_ISSET(S2C(session), WT_CONN_CLOSING))` check up one level we can merge it with the assert on line 123: 
```c
    if (server->threads_working == 0) {
        if (!F_ISSET(S2C(session), WT_CONN_CLOSING)) {

            /* If all the threads are stopped and the queue is empty, background migration is done. */
            if (TAILQ_EMPTY(&server->work_queue))
                WT_ERR(__live_restore_clean_up(session, ctx));

            WT_ASSERT_ALWAYS(session, TAILQ_EMPTY(&server->work_queue),
              "All background migration threads have finished but there is still work to do!");
    }
```

Reply:
neat

Reply:
https://github.com/wiredtiger/wiredtiger/pull/11708/commits/526eeeea9291197855c5c6fe067e9c915c18e9ea